.ci/install_qemu.sh

```diff
@@ -225,7 +223,9 @@ main() {
 			;;
 	esac
 
-	# this variables are used by static-build scripts
+	# Strip "https://" to clone into $GOPATH
+	gopath_qemu_repo=${QEMU_REPO_URL/https:\/\//}
```

Comment:
Nit: This is cute but doesn't anchor the pattern so although it's highly unlikely to be an issue, it would be safer to use something like `sed 's,^https://,,g`

Reply:
Hmm. I kind of like not calling externals and I think we can call this impossible to be an issue rather than just highly unlikely, because that would mean we'd want to have `…https:/…` in a path and if that ever becomes a use case, we took a wrong turn IMO :)